package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "core-data-modelling",
+  "version": "0.0.1",
+  "description": "Get Qlik Associative Engine data model insights.",
+  "author": "QlikTech International AB",
+  "license": "MIT",
+  "scripts": {
+    "build": "cross-env NODE_ENV=production parcel build --no-cache --experimental-scope-hoisting src/index.html",
```

Comment:
Can't remember if I was the one adding `--no-cache` here, but we should probably use it :)